File: Python/4th_Sem_Python_lab/27-03-23/primeCheck.py
#5. Write a Python program to check whether a number is Prime number or not.

num = int(input("Enter a number: "))
isPrime = True # flag variable type of boolean
for i in range(2, num):
    if num % i == 0:
        isPrime = False
        break

if isPrime:
    print(f"{num} is a prime number")
else:
    print(f"{num} is not a prime number")

File: c++/prime_check.cpp
// WAP IN C++ TO CHECK WHETHER A NUMBER IS PRIME OR NOT

#include<iostream>
using namespace std;

int main(){
    int n;
    cout<<"\n\tEnter a number: ";
    cin>>n;
    int i;
    for(i=2; i<n; i++){
        if(n%i==0){
            cout<<"\n\t"<<n<<" is not a prime number";
            break;
        }
    }
    if(i==n){
        cout<<"\n\t"<<n<<" is a prime number";
    }
    return 0;
}